cms/djangoapps/contentstore/features/component.py

```diff
@@ -74,8 +74,7 @@ def see_a_problem_component(step, category):
                 'No problem was added to the unit.')
 
     problem_css = 'li.studio-xblock-wrapper div.xblock-student_view'
-    actual_text = world.css_text(problem_css)
-    assert_in(category.upper(), actual_text)
+    assert_true(world.css_contains_text(problem_css, category.upper()))  # pylint: disable=no-value-for-parameter
```

Comment:
That makes sense; I think it's worth adding a comment here to explain why
it's being done.

On Fri, Jun 12, 2015 at 7:10 AM, Ben Patterson <notifications@github.com>
wrote:

> In cms/djangoapps/contentstore/features/component.py
> <https://github.com/edx/edx-platform/pull/8471#discussion_r32307375>:
>
> > @@ -74,8 +74,7 @@ def see_a_problem_component(step, category):
> >                  'No problem was added to the unit.')
> >
> >      problem_css = 'li.studio-xblock-wrapper div.xblock-student_view'
> > -    actual_text = world.css_text(problem_css)
> > -    assert_in(category.upper(), actual_text)
> > +    assert_true(world.css_contains_text(problem_css, category.upper()))  # pylint: disable=no-value-for-parameter
>
> @sarina <https://github.com/sarina> @clytwynec
> <https://github.com/clytwynec> I took another look this morning. The
> all-caps is how it appears on the page itself. I looked closer at it,
> creating problems, etc, and this is the behavior of the platform:
> [image: image]
> <https://cloud.githubusercontent.com/assets/2004678/8128920/d674bc3e-10d1-11e5-996e-635795f8115b.png>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/edx/edx-platform/pull/8471/files#r32307375>.
>